tests/src/test/scala/it/unibo/scafi/test/functional/stdlib/FieldUtils.scala

```diff
@@ -0,0 +1,205 @@
+package it.unibo.scafi.test.functional.stdlib
+
+import it.unibo.scafi.test.FunctionalTestIncarnation._
+import it.unibo.scafi.test.functional.{ScafiAssertions, ScafiTestUtils}
+import org.scalatest._
+
+
+class FieldUtils extends FlatSpec {
+  import ScafiAssertions._
+  import ScafiTestUtils._
+
+  val Field_Utils = new ItWord
+
+
+  private[this] trait SimulationContextFixture {
+    val net: Network with SimulatorOps = standardNetwork()
+  }
+
+  private[this] trait TestProgram extends AggregateProgram with StandardSensors with BlockG
+
+  Field_Utils should "support min/maxHoodSelectors" in new SimulationContextFixture {
+    // ACT
+    exec(new TestProgram {
+      override def main(): Any = (
+        excludingSelf.minHoodSelector(nbr(mid()))(nbr(mid())),
+        includingSelf.minHoodSelector(nbr(mid()))(nbr(mid())),
+        excludingSelf.maxHoodSelector(nbr(mid()))(nbr(mid())),
+        includingSelf.maxHoodSelector(nbr(mid()))(nbr(mid()))
+      )
+    }, ntimes = fewRounds)(net)
+
+    // ASSERT
+    assertNetworkValues((0 to 8).zip(List(
+      (Some(1),0,Some(4),4), (Some(0),0,Some(5),5), (Some(1),1,Some(5),5),
+      (Some(0),0,Some(7),7), (Some(0),0,Some(7),7), (Some(1),1,Some(7),7),
+      (Some(3),3,Some(7),7), (Some(3),3,Some(6),7), (None,8,None,8)
+    )).toMap)(net)
+
+  }
+
+  Field_Utils should "support anyHood" in new SimulationContextFixture {
+    // ACT
```

Comment:
This test case is quite specific in what it covers. I'd consider its removal if more general test cases are (already?) prepared.